float32 angle_from_robot
float32 distance
float32 angle_from_beacon
bool only_bottom
bool beacon_not_found
bool beacon_angle_conf